Tasks/XamarinLicense/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -8,7 +8,7 @@
   "loc.input.label.email": "Email",
   "loc.input.help.email": "Xamarin account email address.",
   "loc.input.label.password": "Password",
-  "loc.input.help.password": "Xamarin account password.",
+  "loc.input.help.password": "Xamarin account password. Use a Build Variable to encrypt.",
```

Comment:
In my opinion, the added clarity is more important than staying consistent.  I've seen some users get confused about where to create new variables, and it's non-intuitive to enable encryption with the lock icon.